apps/api/src/server/applications/applications.routes.js

```diff
@@ -2,8 +2,8 @@ import express from 'express';
 import { caseAdminOfficerRoutes } from './case-admin-officer/case-admin-officer.routes.js';
 import { caseOfficerRoutes } from './case-officer/case-officer.routes.js';
 import { inspectorRoutes } from './inspector/inspector.routes.js';
-import { sectorRoutes } from './sector/sector.routes.js';
```

Comment:
i'm not sure what's happening here. for some reason the linter wants the sector routes to be below the regions one

Reply:
oh I see! it's alphabetical. well, that's silly